lab-andrew/package.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
This is a good start, you're missing a few critical dev/dependencies see my comments below.